opds_import.py

```diff
@@ -275,7 +276,15 @@ class OPDSImporter(object):
 
     COULD_NOT_CREATE_LICENSE_POOL = (
         "No existing license pool for this identifier and no way of creating one.")
-   
+
+    NAME = ExternalIntegration.OPDS_IMPORT
+    FIELDS = [
+        {
+            "key": Collection.EXTERNAL_ACCOUNT_ID_KEY,
+            "label": _("URL"),
+        }
+    ]
+
```

Comment:
You're right, that is confusing. I can't remember why we wanted it in external_account_id. Maybe we wanted it to be on the Collection instead of the integration.